index.bs

```diff
@@ -5310,9 +5310,7 @@ side=] of [=transform streams=].
  1. Return the result of [=reacting=] to |flushPromise|:
   1. If |flushPromise| was fulfilled, then:
    1. If |readable|.\[[state]] is "`errored`", throw |readable|.\[[storedError]].
-   1. Let |readableController| be |readable|.\[[readableStreamController]].
-   1. If ! [$ReadableStreamDefaultControllerCanCloseOrEnqueue$](|readableController|) is true,
-      perform ! [$ReadableStreamDefaultControllerClose$](|readableController|).
+   1. Perform ! [$ReadableStreamDefaultControllerClose$](|readable|.\[[readableStreamController]]).
```

Comment:
The `ReadableStreamDefaultControllerCanCloseOrEnqueue` check has become redundant, similar to [this change](https://github.com/whatwg/streams/commit/a3e482279576768a602e660e65ea453f9b19b239#diff-ec9cfa5f3f35ec1f84feb2e59686c34dL4997-R4994) in `TransformStreamDefaultControllerTerminate`.